content/md/productivity/get-started-with-the-rules-engine.md

```diff
@@ -11,10 +11,33 @@ The rules engine can radically boost your productivity in the PIM by automating
 
 The rules are defined in a YML file that you can import in the PIM, using a dedicated job profile.   
 
-A rule is defined by a code (required) and you can name it with a label as well. This label can have a value per locale.
-
 This feature is very, like, really powerful, and the use cases are countless. In order to guide you in the rules engine usage, we wrote this article where you will find out what you can use it for and how it works.
 
+# Name your rules
+
+A rule is defined by a code (required) and you can name it with a label as well. This label can have a value per locale.
+Here is the expected format:
+
+```YML
+  camera_copy_name_to_model:
+        priority: 0
+        conditions:
+            - field: family
+              operator: IN
+              value:
+                - camcorders
+            - field: camera_model_name
+              operator: EMPTY
+        actions:
+            - type: copy
+              from_field: name
+              to_field: camera_model_name
+        labels:
+            en_US: 'Copy name to model'
+            fr_FR: 'Copie nom vers modèle'
+```
+
+If no label is defined, we use the code between `[ ]` or `{}`.
```

Comment:
If you do not want to add labels to your rule, leave the labels settings with `[ ]` or `{ }`.

```YML
labels: {  }
```
 or 

```YML
labels: [  ]
```

:::warning
If no label is set, the rule's code will appear into brackets `[ ]` in the PIM user interface.
:::

Reply:
Seen with Pierre, we won't document the [] as it's an unexpected behavior from Php.